luigi_pipeline/seqr_gcnv_loading.py

```diff
@@ -13,6 +14,56 @@
 
 SAMPLE_ID_REGEX = r'(.+)_v\d+_Exome_(C|RP-)\d+$'
 
+EXPECTED_TYPES = {
+    "chr": hl.tstr, 
+    "start": hl.tint32, 
+    "end": hl.tint32, 
+    "name": hl.tstr, 
+    "sample": hl.tstr, 
+    "sample_fix": hl.tstr, 
+    "svtype": hl.tstr, 
+    "GT": hl.tint32, 
+    "CN": hl.tint32,
+    "NP": hl.tint32, 
+    "QA": hl.tint32,
+    "QS": hl.tint32, 
+    "QSE": hl.tint32, 
+    "QSS": hl.tint32, 
+    "ploidy": hl.tint32,
+    "strand": hl.tstr, 
+    "variant_name": hl.tstr, 
+    "ID": hl.tint32, 
+    "rmsstd": hl.tint32,
+    "defragmented": hl.tbool, 
+    "vaf": hl.tfloat64, 
+    "vac": hl.tint32, 
+    "lt100_raw_calls": hl.tbool,
+    "lt10_highQS_rare_calls": hl.tbool, 
+    "PASS_SAMPLE": hl.tbool,
+    "PASS_FREQ": hl.tbool,
+    "PASS_QS": hl.tbool,
+    "HIGH_QUALITY": hl.tbool, 
+    "genes_any_overlap": hl.tstr,
+    "genes_any_overlap_exonsPerGene": hl.tstr, 
+    "genes_any_overlap_totalExons": hl.tint32, 
+    "genes_strict_overlap": hl.tstr,
+    "genes_strict_overlap_exonsPerGene": hl.tint32,
+    "genes_strict_overlap_totalExons": hl.tint32,
+    "genes_CG": hl.tstr, 
+    "genes_LOF": hl.tstr, 
+    "genes_any_overlap_Ensemble_ID": hl.tstr,
```

Comment:
Many of these are columns that are unused upstream. Maybe not worth throwing if an unused column is mis-typed?

Reply:
I think things default ot strings, so its also okay to leave out all the string columns from this mapping too

Reply:
👍